sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -812,6 +812,70 @@ class MapboxMap internal constructor(
     }
   }
 
+  /**
+   * Returns all the leaves (original points) of a cluster (given its cluster_id) from a GeoJsonSource, with pagination support: limit is the number of leaves
+   * to return (set to Infinity for all points), and offset is the amount of points to skip (for pagination).
+   *
+   * Requires configuring the source as a cluster by calling [GeoJsonSource.Builder#cluster(boolean)].
+   *
+   * @param sourceIdentifier GeoJsonSource identifier.
+   * @param cluster Cluster from which to retrieve leaves from
+   * @param limit The number of points to return from the query (must use type [Long], set to maximum for all points). Defaults to 10.
+   * @param offset The amount of points to skip (for pagination, must use type [Long]). Defaults to 0.
+   * @param callback The result will be returned through the [QueryFeatureExtensionCallback].
+   *         The result is a feature collection or a string describing an error if the operation was not successful.
+   */
+  @JvmOverloads
```

Comment:
We have   @JvmOverloads for `getGeoJsonClusterLeaves` but not for the others? What is the difference, the default values?

Reply:
Yes, the other two have no default values.